keyboards/ungodly/launch_pad/readme.md

```diff
@@ -0,0 +1,34 @@
+# Launch Pad
+
+![Launch Pad](https://i.imgur.com/ia9uhRQl.png)
+
+PCB Front             |  PCB Back
+:-------------------------:|:-------------------------:
+![Numpad](https://i.imgur.com/f47ZFZZ.png)  |  ![5x4](https://i.imgur.com/wZH76Pp.png)
+
+**Features:**
+* Onboard atmega32u4 controller, 5V @ 16MHz
+* Kaihl hotswap sockets
+* USB-C connector
+* 18 in-switch RGB LEDs
+* 4 underglow RGB LEDs
+* 100 Ω potentiometer slider
+* (optional) M2 pcb mounting holes
+* (optional) pcb-mount stablizer footprints
+* (optional) pin breakout:
+  * F5, F4, F1, F0, VBUS, GND
+
+Numpad Layout             |  5x4 Layout
+:-------------------------:|:-------------------------:
+![Numpad](https://i.imgur.com/4XvqCBH.jpg)  |  ![5x4](https://i.imgur.com/mwtGnPS.jpg)
+The guidance computer inside the [Launch Pad](https://ungodly.design/products/launch-pad) by Ungodly Design.
+
+* Keyboard Maintainer: [Luis Godinez](https://github.com/luis-Godinez)
+* Hardware Supported: Launch Pad PCB
+* Hardware Availability: [Ungodly.Design](https://ungodly.design/products/launch-pad-pcb)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ungodly/launch_pad:default
+
```

Comment:
Can you please add "clear instructions on how to reset the board into bootloader mode" as per the [PR Checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?